cognee/tests/integration/run_toy_tasks/run_tasks_circular_test.py

```
@@ -0,0 +1,47 @@
+import pytest
+import asyncio
+
+from cognee.modules.pipelines.tasks import Task, TaskConfig
+from cognee.modules.pipelines.exceptions import WrongTaskOrderException
+from cognee.modules.pipelines.operations.run_tasks_v2 import run_tasks_base
+
+
+async def run_and_check_tasks():
+    def number_generator(num, context={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)